### PR TITLE
Fix/DOC-62 - Paymaster fee calculation

### PR DESCRIPTION
**Pull request by uF4No**
Includes fix to calculate the gas fee for the transaction sent to the paymaster.
Improves explanation of what's being built in this tutorial.

### CODE REVIEW

File: docs/dev/tutorials/custom-paymaster-tutorial.md [hunk 1]
```diff
@@ -85,7 +89,7 @@ Note, that only the [bootloader](../developer-guides/contracts/system-contracts.
 
 In this tutorial, we want to charge the user one unit of the `allowedToken` in exchange for her fees being paid by the contract.
 
-The input that the paymaster should receive is encoded in the `paymasterInput`. As described [here](../developer-guides/transactions/aa.md#built-in-paymaster-flows), there are some standardized ways to encode user interactions with paymasterInput. To charge the user, we will require that she has provided enough allowance to the paymaster contract. This is what the `approvalBased` flow can help us with.
+The input that the paymaster should receive is encoded in the `paymasterInput`. As described [in the paymaster documentation](../developer-guides/transactions/aa.md#built-in-paymaster-flows), there are some standardized ways to encode user interactions with paymasterInput. To charge the user, we will require that she has provided enough allowance to the paymaster contract. This is what the `approvalBased` flow can help us with.
```
Comment (githubdoramon) [inline docs/dev/tutorials/custom-paymaster-tutorial.md:92]: just remember to remove the /transaction/ path

File: docs/dev/tutorials/custom-paymaster-tutorial.md [hunk 2]
```diff
@@ -326,13 +333,13 @@ import * as ethers from "ethers";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
 
 // Put the address of the deployed paymaster here
-const PAYMASTER_ADDRESS = "";
+const PAYMASTER_ADDRESS = "<PAYMASTER_ADDRESS>";
 
 // Put the address of the ERC20 token here:
-const TOKEN_ADDRESS = "";
+const TOKEN_ADDRESS = "<TOKEN_ADDRESS";
```
Comment (githubdoramon) [inline docs/dev/tutorials/custom-paymaster-tutorial.md:339]: Missing final >

**Review by githubdoramon — CHANGES_REQUESTED** [review]

File: docs/dev/tutorials/custom-paymaster-tutorial.md (hunk 1 above)
Comment (uF4No) [reply docs/dev/tutorials/custom-paymaster-tutorial.md]: fixed

File: docs/dev/tutorials/custom-paymaster-tutorial.md (hunk 2 above)
Comment (uF4No) [reply docs/dev/tutorials/custom-paymaster-tutorial.md]: fixed

File: docs/dev/tutorials/custom-paymaster-tutorial.md [hunk 3]
```diff
@@ -326,13 +333,13 @@ import * as ethers from "ethers";
...
 
 // Wallet private key
-const EMPTY_WALLET_PRIVATE_KEY = "";
+const EMPTY_WALLET_PRIVATE_KEY = "<EMPTY_WALLET_PRIVATE_KEY>";
```
[the remark this answers is not in the document]
Comment (uF4No) [reply docs/dev/tutorials/custom-paymaster-tutorial.md:344]: fixed

**Review by krofax — APPROVED** [review]

**Pull request merged**